app/models/menu.rb

```diff
@@ -2,6 +2,10 @@ class Menu < ApplicationRecord
   belongs_to :user
   has_many :menu_recipes, dependent: :destroy
   has_many :recipes, through: :menu_recipes, dependent: nil
+
+  def recipes_menu
+    MenuRecipe.where(menu_id: id).map{ |r| [r.recipe, r.recipe_quantity]}
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/SpaceBeforeBlockBraces: Space missing to the left of {.